django_cache_url/__init__.py

```diff
@@ -28,9 +28,12 @@
 # TODO Remove as soon as Django 3.2 goes EOL
 BUILTIN_DJANGO_BACKEND = 'django.core.cache.backends.redis.RedisCache'
 
+DJANGO_REDIS_BACKEND = 'django_redis.cache.RedisCache'
+
 DJANGO_REDIS_CACHE_LIB_KEY = 'redis-cache'
 DJANGO_REDIS_CACHE_BACKEND = 'redis_cache.RedisCache'
-DJANGO_REDIS_BACKEND = 'django_redis.cache.RedisCache' if VERSION[0] < 4 else BUILTIN_DJANGO_BACKEND
+
+DEFAULT_REDIS_BACKEND = DJANGO_REDIS_BACKEND if VERSION[0] < 4 else BUILTIN_DJANGO_BACKEND
```

Comment:
It might be good to set `VERSION[0] < 4` to a constant like:

```python
IS_DJANGO_VERSION_GTE_4 = VERSION[0] >= 4
```

Then when `parse()` is called, add something like the following inside the function:
```python
if IS_DJANGO_VERSION_GTE_4 is True and url.scheme in ('redis', 'rediss', 'hiredis'):
    warnings.warn(
        (
            "django-cache-url is no longer needed since Django version 4 or newer has a native Redids backend built in."
            "See the Django documentation (https://docs.djangoproject.com/en/4.0/topics/cache/#redis) for details."
        ),
        DeprecationWarning
    )
```